examples/framework.rs

```diff
@@ -108,6 +108,7 @@ async fn setup<E: Example>(title: &str) -> Setup {
 
     log::info!("Initializing the surface...");
 
+    #[cfg(not(feature = "webgl"))]
```

Comment:
I think we can probably leave this as it was originally, because examples can use WebGL by allowing secondary backends like GL (e.g. `wgpu::BackendBit::all()` for examples that support DX11 or GL)

Reply:
Tell me if I’m wrong, I think we need WebGPU to be "deactivated", if not, WebGL will never be used.
I’ve never succeeded in using the environment variable in a browser context anyway …

Reply:
I think it should already be deactivated because of how `web` is guarded by `#[cfg(all(target_arch = "wasm32", not(feature = "webgl")))]`, right?

In other words, when `wgpu::BackendBit::all()` and `--features webgl` are used, we will search through the possible backends to see which one can be used. In that case, we should only see that the GL backend (WebGL) is available, so we'll choose that backend.